src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -205,17 +289,48 @@ function PolicyAccountingPage({policy, connectionSyncProgress}: PolicyAccounting
                                     childrenStyles={styles.pt5}
                                 >
                                     <MenuItemList
-                                        menuItems={menuItems}
+                                        menuItems={connectionsMenuItems}
                                         shouldUseSingleExecution
                                     />
+                                    {otherIntegrationsItems && otherIntegrationsItems?.length > 0 && (
+                                        <CollapsibleSection
+                                            title={translate('workspace.accounting.other')}
+                                            wrapperStyle={styles.pr3}
+                                            titleStyle={[styles.textNormal, styles.colorMuted]}
+                                        >
+                                            {otherIntegrationsItems.map((integration) =>
+                                                integration?.icon ? (
+                                                    <MenuItem
+                                                        icon={integration?.icon}
+                                                        iconType={CONST.ICON_TYPE_AVATAR}
+                                                        interactive={false}
```

Comment:
Any specific reason we've set this to false?

Reply:
The same way we want only "pill" button to be clickable when having no connection:
<img width="524" alt="Screenshot 2024-04-26 at 11 11 15" src="https://github.com/Expensify/App/assets/88395093/d77f3266-c4ba-4321-8fcf-a30e0c791d85">
We also want the same behaviour in "other integrations" section:
<img width="488" alt="Screenshot 2024-04-26 at 11 14 34" src="https://github.com/Expensify/App/assets/88395093/9e5755a3-783e-4fcf-b80f-7b775655bbc1">